src/use-target-changes/use-target-changes.ts

```diff
@@ -0,0 +1,158 @@
+import { StimulusUse, StimulusUseOptions } from '../stimulus_use'
+import { method } from '../support/index'
+import { TargetChangesController } from './target-changes-controller'
+
+export interface TargetChangesOptions extends StimulusUseOptions {
+
+}
+
+export class UseTargetChanges extends StimulusUse {
+  controller: TargetChangesController
+  observer: MutationObserver
+  targetElement: Node
+  targets: string[]
+  deprecatedTargets: string[]
+  options: TargetChangesOptions
+  targetSelector: string
+  deprecatedTargetSelector: string
+
+  constructor(controller: TargetChangesController, targets: string[], options: TargetChangesOptions = {}) {
+    super(controller, options)
+
+    this.targetElement = controller.element
+    this.targets = targets
+    this.deprecatedTargets = this.targets.map(target => `${controller.identifier}.${target}`)
+    this.controller = controller
+    this.deprecatedTargetSelector = `data-target`
```

Comment:
Excellent to track both old and new format.
it would be nice to get it from the `schema` as this is configurable (probably not used much)
reference https://github.com/hotwired/stimulus/issues/124#issuecomment-372449543